test/integration/test_speech_to_text_v1.py

```diff
@@ -142,11 +142,13 @@ def test_custom_grammars(self):
             assert get_grammar_result is not None
         else:
             print('Deleting grammar')
-            delete_grammar_result = self.speech_to_text.delete_grammar(
-                customization_id,
-                'test-add-grammar-python'
-                ).get_result()
-            assert delete_grammar_result is not None
+            try:
+                delete_grammar_result = self.speech_to_text.delete_grammar(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'delete_grammar_result' is assigned to but never used](https://app.codacy.com/app/watson-developer-cloud/python-sdk/pullRequest?prid=4545981)